pkg/utils/json.go

```diff
@@ -42,3 +47,51 @@ func removeNaN(hosts []result.Host) []result.Host {
 	}
 	return newHosts
 }
+
+func GetMetics(searchDir string, m *[]metrics.Metrics) error {
+	resultFilePath := path.Join(path.Dir(path.Clean(searchDir)), "result.txt")
+
+	bytes, err := ioutil.ReadFile(resultFilePath)
+	if err != nil {
+		return err
+	}
+
+	r, err := regexp.Compile(`{[^}]*}`)
+	if err != nil {
+		return err
+	}
+
+	// TODO import this from origin; for now it is still a private field
+	marker_name := "cluster_loader_marker"
+	var bm metrics.BaseMetrics
+	for _, jsonBytes := range r.FindAll(bytes, -1) {
+		jsonString := string(jsonBytes)
+		err := json.Unmarshal([]byte(jsonString), &bm)
+		if err != nil {
+			fmt.Printf("cannot unmarshal the line '%v' for BaseMetrics: %v\n", jsonString, err)
+		}
+
+		if marker_name == bm.Marker {
```

Comment:
This isn't generic enough, will we have to compare the `bm.Marker` to a list of `[]marker_names`?

Reply:
Done. Ignored marker at all.